test/libgit2.jl

```diff
@@ -1,9 +1,12 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
-
```

Comment:
this first whitespace is considered significant and will get added back in by the license-headers script